src/main/java/frc/robot/Robot.java

```diff
@@ -15,7 +15,6 @@
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
-import frc.robot.autoRoutines.*;
```

Comment:
This was preventing the code from building when I used `,/gradlew build` so I removed it.

I'm assuming at some point we'll have an autoRoutines namespace but we don't have one yet.